cfpb/cfpb-chart-builder
# WIP: Add unit tests

Adds gulp build tooling + basic unit tests for the `process-csv` script. More documentation coming
 
## Additions

- gulp tasks
- capital framework CSS, for use in the charts demo pages for design accuracy
- unit tests for process-csv
- build pages and run tests with gulp in Travis CI

## Removals

-

## Changes

- update process-csv to export functions for unit tests

## Testing

- check out branch
- run `./setup.sh`
- run `gulp watch` to spin up local server and view sample charts page
- run `gulp test` to run unit tests. 1 is failing, because 🐞  BUG 🐞 

## Review

- @mistergone 
- @marteki 
- @hillaryj 

[Preview this PR without the whitespace changes](?w=0)

## Screenshots


## Notes

-

## Todos

-

## Checklist
